pygac/calibrate_klm.py

```diff
@@ -569,7 +569,7 @@ def calibrate_thermal(raw_counts, prt, ict, space, number_of_data_records, space
 
     iprt = (line_numbers - 1 - offset) % 5
 
-    tprt = np.zeros((float(number_of_data_records)))
+    tprt = np.zeros(number_of_data_records, dtype=np.float)
```

Comment:
Is ``np.float`` appropriate? Or better ``np.float64``?

Reply:
depends what you want :) is `np.float` precise enough for you ? is it ok if it becomes equivalent float128 in the future ?

Reply:
Oops, I thought it would default to float32 :see_no_evil: Then I'll make it a float64 explicitly